src/pages/UserHome.jsx

```diff
@@ -67,14 +73,35 @@ export default function UserHome() {
         </Grid>
         <Grid item sm={3} className={classes.addButton}>
           <Button
-            variant="outlined"
+            variant="contained"
             color="primary"
             component={Link}
             to="/scholarships/new">
             Add Scholarship
           </Button>
         </Grid>
       </Grid>
+      <center>
+        <Typography
+          variant="h5"
+          component="h2"
+          gutterBotton
+          className={classes.noScholarshipsAdded}>
+          No Scholarships Added Yet
+        </Typography>
+      </center>
+      <center>
+        <Typography
+          variant="h10"
+          component="h2"
+          gutterButton
+          color="primary"
+          className={classes.addScholarship}>
+          <p>
+            <a href="more-scholarships">Add A Scholarship</a>
```

Comment:
I don't think at the moment it redirects to `/scholarships/new`, what its doing is refreshing the page.